ext/storage_sources/azure_store/azure_log_system.cpp

```diff
@@ -29,32 +29,32 @@
 #include "azure_log_system.h"
 
 // Constructor for azure_log_system that calls to set the WiredTiger verbosity level.
-azure_log_system::azure_log_system(WT_EXTENSION_API *wt_api, uint32_t wt_verbosity_level)
+azure_log_system::azure_log_system(WT_EXTENSION_API *wt_api, int32_t wt_verbosity_level)
     : _wt_api(wt_api)
 {
     set_wt_verbosity_level(wt_verbosity_level);
 }
 
-// Find Azure Logger level given WiredTiger verbosity level returns Error if not found.
+// Find Azure Logger level given WiredTiger verbosity level returns Warning if not found.
 const Azure::Core::Diagnostics::Logger::Level
 wt_to_azure_verbosity_level(int32_t wt_verbosity_level)
 {
-    if (wt_to_azure_verbosity_mapping.find(wt_verbosity_level) !=
-      wt_to_azure_verbosity_mapping.end())
-        return wt_to_azure_verbosity_mapping.at(wt_verbosity_level);
+    auto res = wt_to_azure_verbosity_mapping.find(wt_verbosity_level);
+    WT_ASSERT(res != wt_to_azure_verbosity_mapping.end());
+    if (res != wt_to_azure_verbosity_mapping.end())
+        return *res;
     else
-        return Azure::Core::Diagnostics::Logger::Level::Error;
+        return Azure::Core::Diagnostics::Logger::Level::Warning;
 }
 
-// Find WiredTiger verbosity level given Azure Logger level returns Error if not found.
+// Find WiredTiger verbosity level given Azure Logger level returns Warning if not found.
 const int32_t
 azure_to_wt_verbosity_level(Azure::Core::Diagnostics::Logger::Level azure_verbosity_level)
 {
-    if (azure_to_wt_verbosity_mapping.find(azure_verbosity_level) !=
-      azure_to_wt_verbosity_mapping.end())
-        return azure_to_wt_verbosity_mapping.at(azure_verbosity_level);
-    else
-        return WT_VERBOSE_ERROR;
+    auto res = azure_to_wt_verbosity_mapping.find(azure_verbosity_level);
+    WT_ASSERT(res != azure_to_wt_verbosity_mapping.end());
+    if (res != azure_to_wt_verbosity_mapping.end())
+        return *res else return WT_VERBOSE_NOTICE;
```

Comment:
This formatting is a bit too creative for my taste ;-)

Reply:
`s_all` did this because I missed a `;`. Its been amended in https://github.com/wiredtiger/wiredtiger/pull/8834/commits/6c9e741783444ee9287cec35cabf4be53807c416 additionally i used assert instead of WT_ASSERT with the absence of a session